haas/ext/switches/migrations/test/e87572b8fa00_create_test_switch_table.py

```diff
@@ -0,0 +1,34 @@
+"""create test switch table
+
+Revision ID: e87572b8fa00
+Revises: 57f4c30b0ad4
+Create Date: 2017-02-08 20:20:44.410779
+
+"""
+
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'e87572b8fa00'
+down_revision = '57f4c30b0ad4'
+branch_labels = ('haas.ext.switches.test',)
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
```

Comment:
Should get rid of these comments (I'm assuming any needed adjusting has been done).